pages/index.tsx

```diff
@@ -24,77 +24,69 @@ import Head from 'next/head';
 import Link from 'next/link';
 import React from 'react';
 import { HiArrowRight } from 'react-icons/hi';
+import type { Theme } from 'theme-ui';
 import { BaseStyles, ThemeProvider } from 'theme-ui';
-import { theme as baseTheme } from '../src';
-import { IndivTheme } from '../src/theme';
+import { theme as baseTheme } from '../src/theme';
+import { scaleAsArray } from '../src/utils';
 
-const Sheet: React.FC<{ theme: IndivTheme; name: string }> = ({
+const Sheet: React.FC<{ theme: Theme; name: string }> = ({
   theme = baseTheme,
   name = 'Base',
 }) => {
   return (
     <ThemeProvider theme={theme}>
-      <Container sx={{ bg: 'muted' }}>
+      <Container>
         <Head>
-          <title>Indiv Theme</title>
+          <title>Bjerk Theme</title>
         </Head>
-        <Box as="header" sx={{ color: 'text' }}>
-          <Container sx={{ pt: 5, pb: [3, 4] }}>
-            <Heading as="h1" variant="title" color="primary">
-              Indiv {name} Theme
-            </Heading>
-            <Grid
-              gap={4}
-              columns="auto auto auto auto"
-              sx={{
-                alignItems: 'center',
-                justifyContent: 'flex-start',
-                fontSize: 2,
-                mt: 3,
-                mb: 2,
-                a: {
-                  fontWeight: 'bold',
-                  textDecoration: 'none',
-                },
-              }}
-            >
-              <NavLink href="https://github.com/indivorg/theme">GitHub</NavLink>
-              <NavLink href="https://npmjs.com/package/@indivno/theme">
-                NPM
-              </NavLink>
-              <Link href="/">Base Theme</Link>
-              <Link href="/web">Web Theme</Link>
-            </Grid>
-          </Container>
-        </Box>
-        <Box
-          as="main"
-          sx={{
-            bg: 'background',
-            color: 'text',
-            borderRadius: 3,
-            py: 3,
-          }}
-        >
-          <Container>
-            <Heading>Text</Heading>
-            <Card>
+        <Flex as="main" sx={{ gap: 4, flexDirection: 'column' }}>
+          <Box as="header" sx={{ color: 'text' }}>
+            <Container>
+              <Heading as="h1" variant="title" sx={{ py: 4 }}>
+                Indiv {name} Theme
```

Comment:
```suggestion
                Bjerk {name} Theme
```

Are we making this the Bjerk theme?

Reply:
Haha, no :)